src/Microsoft.Health.Fhir.SqlServer/Features/Search/Expressions/Visitors/QueryGenerators/SqlQueryGenerator.cs

```diff
@@ -963,7 +964,7 @@ private void WriteIncludeLimitCte(string cteToLimit, SearchOptions context)
                 .AppendLine(" THEN 1 ELSE 0 END AS IsPartial ");
 
             StringBuilder.Append("FROM ").AppendLine(cteToLimit);
-            StringBuilder.Append($"),{Environment.NewLine}");
+            StringBuilder.AppendLine($"),");
```

Comment:
Nit: The $ isn't needed here anymore either.

Reply:
Indeed